### PR TITLE
Update and rename Valve.xml to ValveSoftware.com.xml

### PR DESCRIPTION
**Pull request by koops76**
Redirect to HTTP.

### CODE REVIEW

File: src/chrome/content/rules/developer.valvesoftware.com.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	Other Valve rulesets:
+		- Dota_2.com.xml
+		- Steam.xml
+		- Steam_Community.com.xml
+		- Steam_static.com.xml
+		- Steamgames.com.xml
+-->
+
+<ruleset name="Valve Developer Community">
+	<target host="developer.valvesoftware.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/developer.valvesoftware.com.xml]: See also:
```
valvesoftware.com
www.valvesoftware.com
blogs.valvesoftware.com
list.valvesoftware.com
sources.valvesoftware.com
store.valvesoftware.com

File: src/chrome/content/rules/developer.valvesoftware.com.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!--
+	Other Valve rulesets:
+		- Dota_2.com.xml
+		- Steam.xml
+		- Steam_Community.com.xml
+		- Steam_static.com.xml
+		- Steamgames.com.xml
+-->
+
+<ruleset name="Valve Developer Community">
```
Comment (J0WI) [inline src/chrome/content/rules/developer.valvesoftware.com.xml]: Please add the partial flag and comments for non working hosts.

File: src/chrome/content/rules/developer.valvesoftware.com.xml (hunk 1 above)
Comment (koops76) [reply src/chrome/content/rules/developer.valvesoftware.com.xml]: Which of them work?

Comment (J0WI) [reply src/chrome/content/rules/developer.valvesoftware.com.xml]: `^`, `www` and `list`. You can easily check it in your browser or by command line.

Comment (J0WI) [reply src/chrome/content/rules/developer.valvesoftware.com.xml]: `blogs` is refused, `sources` and `store` have a mismatching cert.

**Comment by koops76** [comment]
Done, `sources` doesn't exist.

**Comment by J0WI** [comment]
It turns out, that `^` and `www` redirect to HTTP. :(

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**